pkgs/by-name/ut/util-linux/package.nix

```
@@ -164,52 +166,48 @@ stdenv.mkDerivation (finalPackage: rec {
-  ++ lib.optionals systemdSupport [ systemd ];
-
-  doCheck = false; # "For development purpose only. Don't execute on production system!"
+  ++ lib.optionals systemdSupport [ systemdLibs ];
 
   enableParallelBuilding = true;
 
-  postInstall =
-    lib.optionalString stdenv.hostPlatform.isLinux ''
-      moveToOutput bin/mount "$mount"
-      moveToOutput bin/umount "$mount"
-      ln -svf "$mount/bin/"* $bin/bin/
-    ''
-    + ''
+  postInstall = ''
+    moveToOutput sbin/nologin "$login"
+    moveToOutput sbin/sulogin "$login"
+    prefix=$login _moveSbin
+    ln -svf "$login/bin/"* $bin/bin/
 
-      moveToOutput sbin/nologin "$login"
-      moveToOutput sbin/sulogin "$login"
-      prefix=$login _moveSbin
-      ln -svf "$login/bin/"* $bin/bin/
-    ''
-    + lib.optionalString withLastlog ''
-      # moveToOutput "lib/liblastlog2*" "$lastlog"
-      ${lib.optionalString (!stdenv.hostPlatform.isStatic) ''moveToOutput "lib/security" "$lastlog"''}
-      moveToOutput "lib/tmpfiles.d/lastlog2-tmpfiles.conf" "$lastlog"
+    ln -svf "$bin/bin/hexdump" "$bin/bin/hd"
+    ln -svf "$man/share/man/man1/hexdump.1" "$man/share/man/man1/hd.1"
 
-      moveToOutput "bin/lastlog2" "$lastlog"
-      ln -svf "$lastlog/bin/"* $bin/bin/
-    ''
-    + lib.optionalString (withLastlog && systemdSupport) ''
-      moveToOutput "lib/systemd/system/lastlog2-import.service" "$lastlog"
-      substituteInPlace $lastlog/lib/systemd/system/lastlog2-import.service \
-        --replace-fail "$bin/bin/lastlog2" "$lastlog/bin/lastlog2"
-    ''
-    + lib.optionalString stdenv.hostPlatform.isLinux ''
+    installShellCompletion --bash bash-completion/*
+  ''
+  + lib.optionalString stdenv.hostPlatform.isLinux ''
+    moveToOutput bin/mount "$mount"
+    moveToOutput bin/umount "$mount"
+    ln -svf "$mount/bin/"* $bin/bin/
+
+    moveToOutput sbin/swapon "$swap"
+    moveToOutput sbin/swapoff "$swap"
+    prefix=$swap _moveSbin
+    ln -svf "$swap/bin/"* $bin/bin/
+  ''
+  + lib.optionalString withLastlog ''
+    ${lib.optionalString (!stdenv.hostPlatform.isStatic) ''moveToOutput "lib/security" "$lastlog"''}
+    moveToOutput "lib/tmpfiles.d/lastlog2-tmpfiles.conf" "$lastlog"
 
-      moveToOutput sbin/swapon "$swap"
-      moveToOutput sbin/swapoff "$swap"
-      prefix=$swap _moveSbin
-      ln -svf "$swap/bin/"* $bin/bin/
-    ''
-    + ''
+    moveToOutput "bin/lastlog2" "$lastlog"
+    ln -svf "$lastlog/bin/"* $bin/bin/
 
-      ln -svf "$bin/bin/hexdump" "$bin/bin/hd"
-      ln -svf "$man/share/man/man1/hexdump.1" "$man/share/man/man1/hd.1"
+  ''
+  + lib.optionalString (withLastlog && systemdSupport) ''
+    moveToOutput "lib/systemd/system/lastlog2-import.service" "$lastlog"
+    substituteInPlace $lastlog/lib/systemd/system/lastlog2-import.service \
+      --replace-fail "/usr/bin/mv" "${lib.getExe' coreutils "mv"}"
+    substituteInPlace $lastlog/lib/systemd/system/lastlog2-import.service \
+      --replace-fail "$bin/bin/lastlog2" "$lastlog/bin/lastlog2"
```

```suggestion
    substituteInPlace $lastlog/lib/systemd/system/lastlog2-import.service \
      --replace-fail "/usr/bin/mv" "${lib.getExe' coreutils "mv"}" \
      --replace-fail "$bin/bin/lastlog2" "$lastlog/bin/lastlog2"
```
No need to do two calls to `substituteInPlace`  for the same file, other than that thank you